backend/app/management/commands/syncdb.py

```diff
@@ -92,21 +95,27 @@ def handle(self, *args, **options):
         # THIS IS JUST FOR PROTOTYPING NEVER EVER EVER EVER IN PRODUCTION do this
         superuser = User.objects.create_superuser('admin', password='adminadmin')
 
-        if not values:
+        if not databases:
             print_header('No data found.')
         else:
-            print_header('Importing these values from the spreadsheet')
-
-            header = values[0]
-            values_as_a_dict = [{header[i]: entry for i, entry in enumerate(row)}
-                                for row in values[1:]]
-
-            for row in values_as_a_dict:
-                print(row)
-                photo = Photo(
-                    title=row["title"],
-                    front_src=row["front_src"],
-                    back_src=row["back_src"],
-                    alt=row["alt"],
-                )
-                photo.save()
+            for m, values in enumerate(databases):
```

Comment:
you can use `zip` here:

`for model_name, values in zip(spreadsheet_ranges, databases):`

and then you don't have to pass the index to get the model_name